docs/blog/posts/2023-08-23-howRenvJupyterKernel.md

```diff
@@ -1,25 +1,34 @@
 ---
 date:
   created: 2023-08-23
+categories:
+    - R
+    - Environments
+authors:
+    - mason
 ---
 
 # How do I export my R environment to JupyterHub?
```

Comment:
For consistency, we should capitalize every word in the titles

Reply:
done